connection.js

```diff
@@ -985,6 +986,7 @@ Connection.prototype.rcpt_ok_respond = function (retval, msg) {
         this.results.add(this, {err: 'rcpt_ok_respond found no transaction'});
         return;
     }
+    if (!msg) msg = this.last_rcpt_msg;
     var rcpt = this.transaction.rcpt_to[this.transaction.rcpt_to.length - 1];
```

Comment:
Here you go, now I'm actually at my computer instead of using my phone...

This is worth a read: http://www.html5rocks.com/en/tutorials/speed/v8/ - the references to the original talk and slides are at the bottom, but this is a good summary of everything discussed in it.